### DISCUSSION TITLE
Why is Parcel 2 ignoring the URL #fragment

### DISCUSSION THREAD

**Discussion by micrology** [parcel-bundler/parcel#6626] [Q&A] [answered]
I have a HTML source file that includes `<a class="nav-link" href="./index.html#about">About</a>`.  When the file is processed through Parcel 2.0.0-beta.2, that line of the file in the `dist` directory  becomes `<a class="nav-link" href="index.html">About</a>`, i.e. the `#about` is omitted.  Is this a bug?  Is there a work around?  I am using the default plugins and `"rm -rf dist/* && parcel build './html/*.html' --no-optimize --no-scope-hoist --detailed-report 1"` for the build.

**Comment by devongovett** [comment, marked as answer]
This was recently fixed in a nightly release

**Reply by micrology** [reply]
Great - it worked.
